src/main/java/com/bootcamp/demo/restapi/BookingsController.java

```diff
@@ -0,0 +1,178 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Booking;
+import com.bootcamp.demo.model.PaymentStatus;
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.model.component.Location;
+import com.bootcamp.demo.model.component.ScooterStatus;
+import com.bootcamp.demo.service.BookingService;
+import com.bootcamp.demo.service.ScooterService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashSet;
+import java.util.Random;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Logger;
+
+
+@RestController
+@RequestMapping(path = "/api/bookings")
+public class BookingsController {
+    private static final ResponseEntity<Object> SUCCESS_RESPONSE = new ResponseEntity<>(HttpStatus.OK);
+    private static final ResponseEntity<Object> FAILURE_RESPONSE = new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+    private static final Logger LOGGER = Logger.getLogger(BookingsController.class.getName());
+    private final BookingService bookingService;
+    private final ScooterService scooterService;
+
+    public BookingsController(BookingService bookingService, ScooterService scooterService) {
+        this.bookingService = bookingService;
+        this.scooterService = scooterService;
+    }
+
+    @PostMapping
+    @ResponseStatus(value = HttpStatus.CREATED)
+    public ResponseEntity<Object> createBooking(@RequestBody final Booking booking) {
+        try {
+            LOGGER.info("Booking created successfully. Update time: " + bookingService.createBooking(booking));

```

Comment:
You could simplify this to sth like (provided that slf4j will be used, since I think this one does not have support for placeholders): `LOGGER.info("Booking created successfully. Update time: ", bookingService.createBooking(booking));`.